src/providers/WorkflowCore.Providers.Azure/README.md

```diff
@@ -2,21 +2,33 @@
 
 * Provides [DLM](https://en.wikipedia.org/wiki/Distributed_lock_manager) support  on [Workflow Core](../../README.md) using Azure Blob Storage leases.
 * Provides Queueing support  on [Workflow Core](../../README.md) using Azure Storage queues.
+* Provides event hub support on [Workflow Core](../../README.md) backed by Azure Service Bus.
 
 This makes it possible to have a cluster of nodes processing your workflows.
 
 ## Installing
 
 Install the NuGet package "WorkflowCore.Providers.Azure"
 
+Using Nuget package console
 ```
 PM> Install-Package WorkflowCore.Providers.Azure
 ```
+Using .NET CLI
+```
+dotnet add package WorkflowCore.Providers.Azure
+```
 
 ## Usage
 
-Use the .UseAzureSyncronization extension method when building your service provider.
+Use the `IServiceCollection` extension methods when building your service provider
+* .UseAzureSyncronization
+* .UseAzureServiceBus
 
 ```C#
-services.AddWorkflow(x => x.UseAzureSyncronization("azure storage connection string"));
+services.AddWorkflow(options => 
+{
+	options.UseAzureSyncronization("azure storage connection string");
+	options.UseAzureServiceBus("service bus connection string", "topic name", "subscription name");
```

Comment:
I not sure about the name of this extension method.  I think we we be explicit of what Azure Service bus is being used for... there is ambiguity between a queue provider and event hubs here.  Then it also leaves to door open to doing an implementation with Azure Event Hubs, which would be a more natural choice for this use case, I think.

Reply:
If we introduce the Workflow-Core abstraction naming in extension methods will look not so nice. E.g. "UseAzureServiceBusEventHub" and for Azure EventHub implementation will look something like "UseAzureEventHubEventHub". I think through documentation user can find out what involves by using this.
Also there is some inconsistency because Redis is using it in the extension method naming and Aws Kinesis not.
Maybe will be nice not to propagate internal abstraction naming to the client and just document it.

Reply:
I do not agree that the explicit naming propagates an internal abstraction.  I think the consumer of the library should be able to control exactly what external provider is used for what.  They should be able to pick and choose between each provider for specific uses.

Reply:
Done